### DISCUSSION TITLE
本地开发经常出现css的chunkId not defined

### DISCUSSION THREAD

**Discussion by yangxiaoguai132** [umijs/umi#8848] [Q&A]
<img width="831" alt="image" src="https://user-images.githubusercontent.com/17949045/182574154-af1e96f4-7d2d-4cb0-baef-0aecb2ba33be.png">

这个问题可以如何解决呢

**Comment by fz6m** [comment]
由于缺乏足够的信息，请提供 [最小重现](https://stackoverflow.com/help/minimal-reproducible-example) 的 demo 帮助排查问题。

**Reply by Ska-cloud** [reply]
umi=3.5.0，经常出现这个问题，开发时出现找不到规律，生产版，在版本升级时或者长时间将页面挂在页面上时也会出现该问题

**Reply by fz6m** [reply]
umi 3 不维护了，建议升级到 umi 4

**Comment by cat-walk** [comment]
一样，遇到这个问题很久了
